ui/src/views/Dashboard.vue

```diff
@@ -1231,9 +1231,24 @@ export default {
       this.migrationUpdate(this.currentApp, "start");
       this.hideStartMigrationModal();
     },
+    cleanError(){
+      // clean error messages fro validation
+      this.error = {
```

Comment:
Vue 2 might experience issues if a reactive object (`error`) is completely replaced like this. Please reset errors like this: `this.error.virtualHost = ""; this.error.adIpAddress = ""; ...`